src/modules/SdnDiag.NetworkController.SF/private/Test-NetworkControllerIsHealthy.ps1

```diff
@@ -0,0 +1,10 @@
+function Test-NetworkControllerIsHealthy {
```

Comment:
Should this call Deployment independent call `Get-SdnNetworkController` or assume SF with `Get-SdnNetworkControllerSF`

Reply:
This one I expect only to be related to SF since this generates error if SF is down. The cert rotation logic for FC still needs to be worked out and is not in scope until later. I plan as things get closer to merging into main, adding hard blocks if function is not supported on FCNC.

In order for this module to do cert rotate for FC, we need to split out rotate of node certs and rest/cluster cert. I started some of that work, but put it on hold while splitting everything into different sub-modules.